cmd/virt-launcher/qemu

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
-if [ "$2" != "release" ] || [ "$3" != "end" ] || [ "$4" != "migrated" ]; then
-	exit
+
+if [ "$2" = "migrate" ] && [ "$3" = "begin" ]; then
+    ncat -U /var/run/kubevirt/migration-hook-socket 2>/dev/null
+    exit 0
```

Comment:
Here, I assume that returning non-zero fails the migration.
Don't we want to do that if `ncat` failed to connect? (or any other error) i.e. `exit $?`